api/src/main/java/org/openmrs/module/eptsreports/reporting/reports/SetupTXNEW.java

```diff
@@ -105,41 +92,23 @@ public String getDescription() {
 	@Override
 	public ReportDefinition constructReportDefinition() {
 		
-		setUpProperties();
-		
 		CohortIndicatorDataSetDefinition dsd = new CohortIndicatorDataSetDefinition();
 		dsd.setName("TX_NEW Data Set");
 		dsd.addParameter(new Parameter("startDate", "Start Date", Date.class));
```

Comment:
Please use the constant `ReportingConstants.START_DATE_PARAMETER` and `ReportingConstants.END_DATE_PARAMETER` rather than creating new parameters.

Reply:
Done